examples/pytorch/graphsage/train_sampling.py

```diff
@@ -9,7 +9,9 @@
 import argparse
 import tqdm
 
-from load_graph import load_reddit, inductive_split
+from torch.cuda import nvtx
```

Comment:
i think it's better to remove nvtx in the example code.

Reply:
Sorry -- removed in 
8ffcbbf